CarmenPaolaInnesBarron/practica7/Practica7.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+"""7_Modelos_del_lenguaje.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1U4zHw46NFaKFY6fcatQSRCOI8QoKmUh2
+"""
+
+import re
+import requests
+from sklearn.model_selection import train_test_split
+from collections import defaultdict, Counter
+import numpy as np
+from itertools import chain
+
+def get_corpus(file):
+    r = requests.get(file)
+    response_list = r.text.split("\n")
+    response_list = response_list[23:] # para deshacernos del inicio que no pertenece al corpus en sí
```

Comment:
Bien ahi el comentario. Aclara bastante aunque recomiendo que lo pongas arriba de la línea para cumplir el estilo de python

```suggestion
    # para deshacernos del inicio que no pertenece al corpus en sí
    response_list = response_list[23:] 
```